src/Core/L10n.php

```diff
@@ -340,7 +341,9 @@ public function tt(string $singular, string $plural, int $count): string
 			$s = $singular;
 		}
 
-		$s = @sprintf($s, $count);
+		// We mute errors here because the translation strings may not be referencing the count at all,
+		// but we still have to try the interpolation just in case it is indeed referenced.
+		$s = @sprintf($s, $count, ...$vars);
```

Comment:
I'd rather try catch it and log these errors to find mismatched Strings and counts

Reply:
In this case, we expect mismatched strings and counts because the count is an obligatory parameter to this call, but it doesn't have to be referenced in the string. So logging these errors would just fill your log with expected error messages, it's unworkable.

Reply:
I think I understood it.. :D

Reply:
Allow me to make it clearer: in the `L10n->t()` case, we have no mandatory interpolated value, it directly depends on the string itself so we can output `sprintf()` errors.

However, in the `L10n->tt()` case, we have to pass the count to be able to pick the relevant plural string. Then we have the count at hand, and the translation string may or may not reference to it. Since we can't guess which, we have to call `sprintf()` with at least the count, which will lead to expected notice messages with the strings that do not reference it.